core/src/main/java/com/opensymphony/xwork2/ognl/OgnlDefaultCache.java

```diff
@@ -30,10 +30,10 @@
 public class OgnlDefaultCache<Key, Value> implements OgnlCache<Key, Value> {
 
     private final ConcurrentHashMap<Key, Value> ognlCache;
-    private final AtomicInteger cacheEvictionLimit = new AtomicInteger(25000);
+    private final AtomicInteger cacheEvictionLimit = new AtomicInteger();
```

Comment:
These init values are inconsequential as far as I can tell as they are immediately overridden by the constructor

Reply:
The overriding is a fair point - I'll move the initialisation of the whole field into the constructor